infra/README.md

```diff
@@ -4,19 +4,21 @@ Welcome. This will auto-provision using [Terraform] solely the CI/CD pipeline fo
 The following are the main resources that will be set up for you.
 
 * [Cloud Build]
+* [Cloud Build Triggers]
 * [Cloud Deploy]
 
 ## Getting started
 
-This assumes that you have the following services already existing in your current Google Cloud Platform.
-If your project does fit these requirements, Continue to the [CI/CD documentation](./environments/dev/README.md) to learn more, 
-otherwise follow the pre-requisites before continuing.
+These instructions assume that you have an existing operational [Cloud Run Service] and the following resources:
 
 * [Artifact Registry] for managing container images
 * [Cloud Run] for scalable serverless apps
 * [Cloud Firestore] for scalable serverless databases
 * [Secret Manager] for managing project secrets storage
 
+If your project does meet this requirement, continue to the [CI/CD documentation](./environments/dev/README.md) to learn more, 
```

Comment:
I moved this so it's after the resources, but I'm confused as it's saying jump to this other documentation. I think this sentence needs to be refactored.